packages/cryptoplease/lib/features/investments_settings/module.dart

```diff
@@ -0,0 +1,23 @@
+import 'package:flutter/material.dart';
+import 'package:nested/nested.dart';
+import 'package:provider/provider.dart';
+
+import '../../di.dart';
+import 'src/data/repository.dart';
+
+export 'src/data/repository.dart';
+
+class InvestmentSettingsModule extends SingleChildStatelessWidget {
```

Comment:
The feature should rather be called `investments` and should provide widgets like settings toggle and "My portfolio" list.

The main idea of features/modules is to make them independent and encapsulated. With this implementation, the logic is spread across the whole app: balances state, investments screen, investment settings feature.